habitat_sim/utils/data/dataextractor.py

```diff
@@ -0,0 +1,283 @@
+import collections
+import copy
+import math
+import os
+
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from torch.utils.data import Dataset
+
+import habitat_sim
+import habitat_sim.bindings as hsim
+from examples.settings import make_cfg
```

Comment:
We should not be pulling something in from examples into the core of habitat-sim.